andela/ah-lobos-frontend
# #166789831 Users should be presented with a reset password link

#### What does this PR do?

 - Users should be presented with a reset password link

#### Description of Task to be completed?

  - A user should be presented with a button and or link to reset their passwords on the 
 SignUp/Register component 

#### How should this be manually tested?

- If you forgot the password you may click on forgot password link on the login page. You will be redirected to the page to provide your email and after that, you will get a link to reset password in your email. When you click on that link you will get a page where you may reset a new password and confirm it

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [#166789831](https://www.pivotaltracker.com/story/show/166789831)

#### Screenshots (if appropriate)

- add a forgot password link

![image](https://user-images.githubusercontent.com/48624152/63834066-b5808980-c974-11e9-8dcc-37292479b58c.png)

- set email 

![image](https://user-images.githubusercontent.com/48624152/63859554-e4642300-c9a7-11e9-8f26-10602b4bd38a.png)

- set new password and confirm password

![image](https://user-images.githubusercontent.com/48624152/63859581-ef1eb800-c9a7-11e9-9295-1b127ec32037.png)

#### Questions:

- N/A
